ZenJourney/app/src/main/java/ca/uwaterloo/cs/HomePage.kt

```diff
@@ -156,6 +158,10 @@ fun HomePage(appState: AppState) {
             )
         )
     }
+    if (!isCustomDefaultNotificationSet) {
+        // default notification time (if not custom set) is 12PM
+        scheduleNotification(appState.context, 19, 29)
```

Comment:
why is this specifically 19 and 20?

Reply:
oops, forgot to change this back, ty for catching that